### ISSUE TITLE
Can't replicate metrics plus more notes

### ISSUE THREAD

**Issue by Vynikal** [TimFelixBeyer/weakly-supervised-cad-retrieval#6]
Hello,
I am unable to replicate metrics in the paper on page 11. These are as follows:
0.48 0.66 0.54 0.53 0.23
By using the pretrained 'seen.pth' checkpoint, I am only getting these (on a test split):
45.791 63.737 51.153 52.571 21.724

I also trained my own model. On that note I found this method significantly worsens the resulting metrics:
https://github.com/TimFelixBeyer/weakly-supervised-cad-retrieval/blob/8ecb08b52dccfc9606f377a1cbb6d5c0e5880927/instance_retrieval/network/train.py#L94
I haven't seen it mentioned in the paper. At first I thought it's a kind of augmentation method, but it hardly improves anything.
After removing it, I was able to get:
43.941 60.315 50.071 50.982 20.197

Also, I was able to create my own perceptual dataset from the top-1 similarity metrics. That differed quite significantly from the one provided in the repo.

I am mostly interested in reliable retrieval from laser scanned clouds, thus top-1 metric is important for me. I am aware that Scan2CAD annotations are quite imprecise themselves, so I am trying to use the new LASA dataset, which contains precise 1:1 annotations. At this time I am trying to get render-and-compare retrieval as reliable as possible, then maybe learning an embedding space for faster (but still reliable) retrieval.

I am aware that you probably stopped working in this field. Nonetheless, I would appreciate your insights to anything I wrote. Thanks.
